static/templates/recent_topic_row.hbs

```diff
@@ -17,6 +17,18 @@
         <i class="fa fa-check-circle on_hover_topic_read recipient_bar_icon" data-stream-id="{{stream_id}}" data-topic-name="{{topic}}" title="{{t 'Mark as read' }}" role="button" tabindex="0" aria-label="{{t 'Mark as read' }}"></i>
     </td>
     <td class='recent_topic_users'>
+        <ul class="recent_avatars">
+            {{#if count_senders}}
+            <li class="recent_avatars__item">
```

Comment:
Why the `__` naming here?

Reply:
Yeah, I forgot to refactor this section, done now. (This is from the tutorial I followed to implement this)